addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -19,6 +19,9 @@ async function twoDeleteForward(editor) {
     await deleteForward(editor);
     await deleteForward(editor);
 }
+async function insertCharacter(editor, character) {
```

Comment:
Why not use the existing `insertText` and give it a single character ? We do that in lots of places already.

Reply:
Because I need the event to travel. I need an input, not just the insertText command.

Reply:
I'm not talking about the `insertText` command, I'm talking about the test util function.
See https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/editor/odoo-editor/test/utils.js#L469

Reply:
Done.